src/web/screens/routeUtils.tsx

```diff
@@ -21,15 +21,6 @@ export const makePrivateRoute = (route: PortalRoute | RouteObject): PortalRoute
   const privateRoute: PortalRoute = {
     ...route,
     element: <PrivateRoute>{route.element}</PrivateRoute>,
-    loader: route.loader
-      ? async (args: LoaderFunctionArgs) => {
-          if (!keycloak.authenticated) {
```

Comment:
There was no need to wait until the router setup - just always load the token if available.